### ISSUE TITLE
Use Ctrl+K rather than Ctrl to focus on a "Ask Supabase AI a question"

### ISSUE THREAD

**Issue by jacob-8** [supabase/supabase#15475]
# Improve documentation

## Describe the problem

Every time I try to select a portion of an AI answer to copy with `Ctrl+C`, my selection disappears and focus is placed on the input box as soon as I press `Ctrl`. It's maddening.

## Describe the improvement

Please change the keyboard shortcut to `Ctrl+K` which makes sense as that is the original shortcut that pulled up the box in the first place.

## Additional context

None, except the AI feature is AMAZING and a GAME-CHANGER. So much better than searching docs. I love being able to just ask my questions. Here's my latest which came as a result of the docs at https://supabase.com/docs/guides/auth/auth-helpers/sveltekit#creating-a-supabase-client just telling me `content-range` was needed, but not why. No big deal, I can just ask:

<img width="381" alt="image" src="https://github.com/supabase/supabase/assets/7559478/64a76423-d8a2-44b0-b2d9-c528340851d4">

If others don't adopt this as well, they'll quickly become less attractive.

**Comment by gregnr** [comment]
Great feedback @jacob-8. Looks like this is an unintended side-affect of our auto input focus logic:
https://github.com/supabase/supabase/blob/cb28597e77f71f7f9939ec8a537d9c5b439314ec/packages/ui/src/components/Command/Command.utils.tsx#L409-L420

Would love to see a PR to exclude modifier keys (ctrl, alt, etc) from this logic!

**Comment by pedrovrima** [comment]
I can take care of this ☺️

**Comment by jacob-8** [comment]
> I can take care of this ☺️

Great! For me just Ctrl but Mac probably needs the meta key and others may want alt. I don't see shift being a problem as that's usually the beginning of a sentence.

Does something like this work?

function onKeyDown(e: KeyboardEvent) {
  if (!e.ctrlKey && !e.altKey && !e.metaKey) {
    input?.focus();
  }
}

**Comment by Taiwrash** [comment]
> I can take care of this ☺️

Are you on this or I should look into this with you?

**Comment by pedrovrima** [comment]
Hey @Taiwrash! Thanks for your offer - the PR was ready and I just wanted to do a last review of the code before submitting it.

@jacob-8 - I also had to add another event lister for keyup. Even though this solution avoids the focus on CTRL, as soon as you hit the C for copy it would focus and you wouldn't be allowed to copy the text as well.

I also decided to include all modifier keys except shift - to be safer not to miss anything. Tab, for example, is an important one for a11y.
